addon/globalPlugins/EnhancedDictionaries/guiHelper.py

```diff
@@ -170,7 +172,16 @@ def makeSettings(self, settingsSizer):
 				label=_("&import entries from default profile dictionary")
 			).Bind(wx.EVT_BUTTON, self.onImportEntriesClick)
 
-		sHelper.addItem(bHelper)
+			sHelper.addItem(bHelper)
+
+			profile = config.conf.getActiveProfile()
+			if profile.name != __("Normal configuration"):
+				self.keepUpdatedCheckBox = wx.CheckBox(
+					self, label=_("&Sync entries with default profile dictionary.")
+				)
+				savedCheckboxValue = profileConfigurationHelper.getSavedCheckboxValueForProfile()
+				self.keepUpdatedCheckBox.SetValue(savedCheckboxValue)
```

Comment:
Variable name suggestion
```suggestion
				savedKeepDictionaryUpdatedCheckboxValue = profileConfigurationHelper.getSavedKeepDictionaryUpdatedCheckboxValueForProfile()
				self.keepUpdatedCheckBox.SetValue(savedKeepDictionaryUpdatedCheckboxValue)
```